### ISSUE TITLE
Wrong Webview View tab order / keyboard navigation order

### ISSUE THREAD

**Issue by 0x4ab** [microsoft/vscode#203909]
Does this issue occur when all extensions are disabled?: No (because it is extension-related)

- VS Code Version: 1.85.2 (Universal)
- OS Version: 14.3 (23D56)

Steps to Reproduce:

1. Download, open, compile and run (start debugging) **webview-view-sample** from  [vscode-extension-samples](https://github.com/microsoft/vscode-extension-samples)
2. In the _extension development_ instance open Explorer Side Bar
3. Bring focus to "Calico Colors" header (click), but keep it expanded (click may be needed)
4. Press "tab" to move the focus to the next element ("Clear colors" button is focussed)
5. Press "tab" again to move the focus to the webview view panel
6. Observe that focus has moved to somewhere else (typically the document tab)
   Note: The Webview panel is accessible, but somewhere else in the tab order as the hosting iframe is located  somewhere else in the page hierarchy; This is confusing and not accessible

#### Full version info
```
Version: 1.85.2 (Universal)
Commit: 8b3775030ed1a69b13e4f4c628c612102e30a681
Date: 2024-01-18T06:40:32.531Z
Electron: 25.9.7
ElectronBuildId: 26354273
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Darwin arm64 23.3.0
```
